nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -105,7 +105,7 @@ let
   upstreamConfig = toString (flip mapAttrsToList cfg.upstreams (name: upstream: ''
     upstream ${name} {
       ${toString (flip mapAttrsToList upstream.servers (name: server: ''
-        server ${name} ${optionalString server.backup "backup"};
+        server ${name} ${optionalString server.backup "backup"} ${optionalString (server.extraOptions != []) "${concatStringsSep " " server.extraOptions}"};
```

Comment:
Shouldn't we migrate server.back to this option?

Reply:
It can be done. I wanted to do PR with minimal changes, so as not to break existing configurations.

Reply:
I was also unable to rename the parameter:
```
    (mkRenamedOptionModule [ "services" "nginx" "upstreams" "" "servers" "backup" ] [ "services" "nginx" "upstreams" "" "servers" "extraOptions" ])
```
```
error: The option `services.nginx.upstreams' in module `/home/user/nixpkgs/nixos/modules/services/web-servers/nginx/default.nix' would be a parent of the following options, but its type `attribute set of (submodule)' does not support nested options.
        - option(s) with prefix `services.nginx.upstreams.""' in module `/home/lafiel/nixpkgs/nixos/modules/services/web-servers/nginx/default.nix'
(use '--show-trace' to show detailed location information)
building Nix...
```